src/build.py

```diff
@@ -287,6 +287,13 @@ def JavaBinDir():
 BUILDBOT_BUILDNUMBER = os.environ.get('BUILDBOT_BUILDNUMBER', None)
 BUILDBOT_BUILDERNAME = os.environ.get('BUILDBOT_BUILDERNAME', None)
 
+if IsMac() and IsBuildbot():
+  # Experimental temp fix for crbug.com/829034 stdout write sometimes fails
+  import fcntl
+  from os import O_NONBLOCK
+  fd = sys.stdout.fileno()
+  flags = fcntl.fcntl(fd, fcntl.F_GETFL)
+  fcntl.fcntl(fd, fcntl.F_SETFL, flags & ~O_NONBLOCK)
```

Comment:
Nit: See it might make more sense to say `from fcntl import (fcntl, F_GETFL, F_SETFL)` because that's lot of `fcntl` in a short space